### PR TITLE
feat(github): split repo and org webhooks to separate files

### PR DESCRIPTION
**Pull request by maxamel**
## Description

This PR splits multiple vcs webhooks into separate files so each one appears as new check violation.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my feature, policy, or fix is effective and works
- [x] New and existing tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by lirshindalman — APPROVED** [review]

**Review by Eliran-Turgeman — APPROVED** [review]

File: checkov/github/dal.py [hunk 1]
```diff
@@ -107,7 +109,9 @@ def get_repository_webhooks(self) -> dict[str, Any] | None:
     def persist_repository_webhooks(self) -> None:
         repository_webhooks = self.get_repository_webhooks()
         if repository_webhooks:
-            BaseVCSDAL.persist(path=self.github_conf_file_paths["repository_webhooks"], conf=repository_webhooks)
+            for idx, item in enumerate(repository_webhooks):
+                path = str(self.github_conf_file_paths["repository_webhooks"])
+                BaseVCSDAL.persist(path=path.replace(".json", str(idx) + ".json"), conf=[item])    # type: ignore
```
Comment (lirshindalman) [inline checkov/github/dal.py:114]: can you add some logs here for `path=path.replace(".json", str(idx) + ".json")` ?

**Comment by nimrodkor** [comment]
What about repo webhooks?

File: checkov/github/dal.py (hunk 1 above)
Comment (maxamel) [reply checkov/github/dal.py:114]: We have the logs inside the persist method, so we'll know what we're writing and to where

**Review by nimrodkor — APPROVED** [review]
Can you share an example of the new output?

**Comment by maxamel** [comment]
Example output: 
```
`Check: CKV_GITHUB_7: "Ensure GitHub repository webhooks are using HTTPS"
	FAILED for resource: /github_conf/repository_webhooks2.json.*./github_conf/repository_webhooks2.json.CKV_GITHUB_7
	Severity: MEDIUM
	File: /repository_webhooks2.json:10-13
	Guide: https://docs.bridgecrew.io/docs/ensure-github-organization-and-repository-webhooks-are-using-https
```

		10 |         "config": {
		11 |             "content_type": "form",
		12 |             "insecure_ssl": "0",
		13 |             "url": "http://howdy.co.com"
		
```
Check: CKV_GITHUB_7: "Ensure GitHub repository webhooks are using HTTPS"
	FAILED for resource: /github_conf/repository_webhooks1.json.*./github_conf/repository_webhooks1.json.CKV_GITHUB_7
	Severity: MEDIUM
	File: /repository_webhooks1.json:10-13
	Guide: https://docs.bridgecrew.io/docs/ensure-github-organization-and-repository-webhooks-are-using-https
```

		10 |         "config": {
		11 |             "content_type": "form",
		12 |             "insecure_ssl": "0",
		13 |             "url": "http://dimdim.com"`

**Review by marynaKK — APPROVED** [review]

**Pull request merged**